### DISCUSSION TITLE
Mensagens do Toastr não são exibidas

### DISCUSSION THREAD

**Discussion by AleDinizRJ** [balta-io/7187#44] [Perguntas e Respostas] [answered]
Olá Balta, boa noite!

Eu já fiz várias pesquisas para analisar possíveis conflitos, fiz todas as orientações que pediam na internet, mas continuo sem as mensagens. Eu estou utilizando o Angular 12.

Desde já obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @AleDinizRJ, como vai?
Por favor, utilize as mesmas versões que usamos no curso, assim não tem erro!
